viewing_party/party.py

```diff
@@ -0,0 +1,131 @@
+def create_movie(title, genre, rating):
+    m = dict();
```

Comment:
Variable names like `m` are ok for counter variables, but try to provide more meaningful variable names for other use cases.  In this case you might consider naming it `movie`.
Python allows but does not encourage semicolons.
```suggestion
    movie = dict()
```